src/euphorie/client/browser/organisation.py

```diff
@@ -336,6 +341,29 @@ def organisation(self):
             self.token_value["organisation_id"]
         )
 
+    @property
+    @memoize_contextless
+    def inviter(self):
+        if not self.token_value:
+            return None
+        userid = self.token_value["userid"]
+        try:
+            user = self.sqlsession.query(Account).filter(Account.id == userid).first()
+        except Exception:
+            logger.warning("Unable to fetch account for username:")
+            logger.warning(userid)
+            return None
+        return user
+
+    @property
+    def inviter_fullname(self):
+        user = self.inviter
+        if user is None:
+            return ""
+        if not (user.last_name or user.first_name):
+            return user.loginname
+        return " ".join(filter(None, (user.first_name, user.last_name)))
```

Comment:
```suggestion
        return user.title
```
See: https://github.com/euphorie/Euphorie/blob/6aad346174f2a57a1bc2e9bc340862658bc62b77/src/euphorie/client/model.py#L516-L526